data/bboxes.js

```diff
@@ -0,0 +1,2954 @@
+var bboxes = {
```

Comment:
Yeah we can put these files in the database. I haven't look at how you're using this but if you have to load all the data anyways, it's really no different having it here than having it there, but for any datasets where you're just going to hit one endpoint it makes sense to only load what's necessary. I think for updating the data, it might be slightly easier to do it through the api, but not terribly so.